dockerfiles/phone-number-privacy/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:10
+WORKDIR /celo-phone-number-privacy/
+
+COPY packages/phone-number-privacy/signer signer
+COPY packages/phone-number-privacy/common common
+
+WORKDIR /celo-phone-number-privacy/signer
+COPY yarn.lock ./
+
+RUN yarn install --frozen-lockfile --network-timeout 100000 && yarn cache clean
+
+ENV NODE_ENV production
```

Comment:
where will we add the instructions for what to set other ENV variables here?

Reply:
Added to the readme